src/supaplex.c

```diff
@@ -1505,6 +1510,10 @@ void runAdvancedOptionsRootMenu()
 
 int main(int argc, char *argv[])
 {
+#ifdef __NDS__
+    nitroFSInit(NULL);
+#endif

```

Comment:
I'd rather have this in the `initializeSystem` function in `system.c` and keep this file unaware of the different supported platforms as much as possible.

Unfortunately, the `__PSP__` stuff is the only thing I can't get rid of 😢